package.json

```diff
@@ -21,7 +22,7 @@
   "scripts": {
     "predeploy": "npm run build",
     "deploy": "gh-pages -d build",
-    "start": "react-scripts start",
+    "start": "react-scripts start --host 0.0.0.0",
```

Comment:
what does this do? a quick google search doesn't give an answer

Reply:
Bind the server to all ip's so it could be reached from WAN